tests/testdata/programs/ts/rdsmod/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as rds from "@pulumi/rds";
+
+
+const testrds = new rds.Module("test-rds", {
+    identifier: "test-rds-module",
+    engine: "mysql",
+    instance_class: "db.t3.micro",
+    allocated_storage: 20,
+    db_name: "testrdsmoduledatabase",
+    username: "pulumipus",
+    password: "hawaii",
```

Comment:
This would be a good chance to use another resource to generate a password and pass it in as a reference.